app/content/zones/warTemple.ts

```diff
@@ -6336,9 +6336,9 @@ const f2_0x0: AreaDefinition = {
         {status: "closed", id: "warTemple:2:0x0-door-0", x: 96, y: 240, type: "door", style: "stone", d: "down"},
         {status: "closedEnemy", id: "warTempleRandomizerExit", x: 176, y: 16, type: "door", style: "stoneDownstairs", targetZone: "overworld", targetObjectId: "warTempleEntrance", d: "up", linked: true, logicKey: "isRandomizer"},
         {status: "hiddenEnemy", id: "warTemple:2:0x0-astralProjection-0", spirit: true, x: 64, y: 64, type: "chest", lootType: "astralProjection", lootLevel: 1},
-        {status: "normal", id: "warTempleBoss", spirit: true, x: 48, y: 156, type: "boss", enemyType: "flameIdol", lootType: "peachOfImmortality", lootLevel: 1, d: "down", params: {}},
-        {status: "normal", id: "warTempleBoss", spirit: true, x: 112, y: 76, type: "boss", enemyType: "stormIdol", lootType: "peachOfImmortality", lootLevel: 1, d: "down", params: {}},
-        {status: "normal", id: "warTempleBoss", spirit: true, x: 176, y: 156, type: "boss", enemyType: "frostIdol", lootType: "peachOfImmortality", lootLevel: 1, d: "down", params: {}},
+        {status: "normal", id: "warTempleBoss", spirit: true, x: 48, y: 168, type: "boss", enemyType: "flameIdol", lootType: "peachOfImmortality", lootLevel: 1, d: "down", params: {}},
+        {status: "normal", id: "warTempleBoss", spirit: true, x: 120, y: 92, type: "boss", enemyType: "stormIdol", lootType: "peachOfImmortality", lootLevel: 1, d: "down", params: {}},
+        {status: "normal", id: "warTempleBoss", spirit: true, x: 178, y: 168, type: "boss", enemyType: "frostIdol", lootType: "peachOfImmortality", lootLevel: 1, d: "down", params: {}},
```

Comment:
It looks like my recent changes from main our showing up in this PR for some reason, are you seeing this as well? I doubt it matters much, but these shouldn't be showing up as changed files and make it slightly harder to review.